<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spaghetti::nodes::values::characteristic_curve::EditorWindow</class>
 <widget class="QDialog" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>1042</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Characteristic Curve Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
   <item>
    <widget class="spaghetti::nodes::values::characteristic_curve::EditorWidget" name="editor"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
     <item>
      <widget class="QToolBox" name="options">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="xAxisPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>205</height>
         </rect>
        </property>
        <attribute name="label">
         <string>X Axis</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="xAxisNameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="xAxisName">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="xAxisTicksBox">
           <property name="title">
            <string>Ticks</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QSpinBox" name="xAxisMajorTicks">
              <property name="minimum">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="xAxisMinorTicks"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QGroupBox" name="xAxisRangeBox">
           <property name="title">
            <string>Range</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QDoubleSpinBox" name="xAxisMinimum">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="xAxisMaximum">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="yAxisPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>205</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Y Axis</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="yAxisNameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="yAxisName">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="yAxisTicksBox">
           <property name="title">
            <string>Ticks</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QSpinBox" name="yAxisMajorTicks">
              <property name="minimum">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="yAxisMinorTicks"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QGroupBox" name="yAxisRangeBox">
           <property name="title">
            <string>Range</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QDoubleSpinBox" name="yAxisMinimum">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yAxisMaximum">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="csvTemplatesPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>258</width>
          <height>270</height>
         </rect>
        </property>
        <attribute name="label">
         <string>CSV/Templates</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="csvBox">
           <property name="title">
            <string>CSV</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="csvImport">
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="csvExport">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="csvToClipboard">
              <property name="text">
               <string>To Clipboard</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="csvFromClipboard">
              <property name="text">
               <string>From Clipboard</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="templatesBox">
           <property name="title">
            <string>Templates</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="templateTypeLabel">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="templateType">
              <item>
               <property name="text">
                <string>EasyIn</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="templateStepsLabel">
              <property name="text">
               <string>Steps</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="templateSteps"/>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="templateGenerate">
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="seriesBox">
       <property name="title">
        <string>Series</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="seriesTable">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="live">
            <property name="text">
             <string>Live</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>spaghetti::nodes::values::characteristic_curve::EditorWidget</class>
   <extends>QGraphicsView</extends>
   <header>nodes/values/characteristic_curve/editor_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
